boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_shared.dts

```diff
@@ -118,12 +77,9 @@ leds:	leds {
 		led0 = &rgb1_red;
 		led1 = &rgb1_green;
 		led2 = &rgb1_blue;
-		led3 = &rgb2_red;
-		led4 = &rgb2_green;
-		led5 = &rgb2_blue;
-		led6 = &led1_blue;
-		led7 = &led2_green;
-		led8 = &led3_green;
+		led3 = &led1_blue;
+		led4 = &led2_green;
+		led5 = &led3_green;
```

Comment:
Hmm, why have you only removed the NET core RGB LED here in this _shared.dts file?

Reply:
I have added them back, but this may be a bit misleading as well as we have given these LEDs explicitly to NET. What we could do is remove them all, but then there will be less flexibility in assigning the LEDs in alternative manners.